lib/storage/src/content_manager/toc/point_ops.rs

```diff
@@ -176,7 +176,38 @@ impl TableOfContent {
         group_by
             .execute()
             .await
-            .map(|groups| GroupsResult { groups })
+            .map(|mut groups| GroupsResult {
+                groups: groups.pop().unwrap_or_default(),
```

Comment:
In this case, `execute` should always return one element. Would be preferable to return an error or use `unwrap()` instead of `unwrap_or_default()`?